.github/workflows/release.yml

```diff
@@ -36,12 +36,16 @@ jobs:
           KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
           KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
           KEY_STORE_PASSWORD: ${{ secrets.KEY_STORE_PASSWORD }}
-        run: ./gradlew publishReleaseApk
+        run: ./gradlew publishReleaseBundle
 
       - name: Publish app to download.kiwix.org
         env:
+          KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
```

Comment:
I don’t understand why we need this now. This ruke should not have changed?!

Reply:
I thought we need an alias name. Ok, I'll remove that 👍 .